tools/run_tests/performance/scenario_config.py

```diff
@@ -832,8 +832,20 @@ def scenarios(self):
         rpc_type='STREAMING', client_type='SYNC_CLIENT', server_type='SYNC_SERVER',
         server_language='c++', async_server_threads=1)
 
-    # TODO(ddyihai): Investigate why when async_server_threads=1/CPU usage 340%, the QPS performs
-    # better than async_server_threads=0/CPU usage 490%.
+    yield _ping_pong_scenario(
```

Comment:
Should QPS for 1M payload use unconstrained model which is consistent with other QPS test?
But it seems other scenarios on this dashboard are using constrained model.